### PR TITLE
To-Do list: Add remove functionality

### PR DESCRIPTION
**Pull request by BertrandConxy**
# To-Do list: Add-remove functionality

In this branch, I have done the following:
- Removed all hardcoded array of activities
- Created a new javascript file for adding and removing classes
- Created another js file to handle removing the list item from the UI
- Created a function to edit task description when the user clicks on it
- Activities array is stored in local storage and is updated continuously.

### CODE REVIEW

File: src/js/add-remove-localstorage.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+}
+
+export class StoreActivity {
+  constructor(index, description, complete) {
+    this.index = index;
+    this.description = description;
+    this.complete = complete;
+  }
+}
```
Comment (D4nielJ) [inline src/js/add-remove-localstorage.js:37]: - [x] Let's make sure that whenever an activity is marked as completed the correct state is stored on local storage.
> All changes to the To Do List should be saved in local storage.

no-completed

File: src/js/UI-remove-task.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default class RemoveActivities {
```
Comment (D4nielJ) [inline src/js/UI-remove-task.js:1]: - [x] Working on new milestones should not remove previous functionality added, let's make sure that the clear all completed button keeps working as expected.

> Implement a function for the "Clear all completed" button (use filter() method).

no-completed

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @BertrandConxy, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- I can add new todos!
- I can edit the todos and remove them :clap: :fireworks: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thank you @D4nielJ  for your reviews. I am going to implement that. However, I  think those were the requirement for the last part of this project which is the interactive list. May you check on that again please?

**Comment by D4nielJ** [comment]
> Thank you @D4nielJ for your reviews. I am going to implement that. However, I think those were the requirement for the last part of this project which is the interactive list. May you check on that again, please?

I'm very sorry about this. You are correct. These are requirements from the next milestone. Please ask for a new review so the next reviewer can approve your project and unblock you.

**Review by mikethreels — APPROVED** [review]
Hi Bertrand, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thank you @mikethreels  and @D4nielJ, I appreciate your reviews.

**Pull request merged**